lib/cursor.js

```diff
@@ -18,250 +18,250 @@ class Cursor {
   }
 
   // Attach schema
-  //   schema, // object, schema
-  //   category // string, schema name
-  // Returns: this instance
+  //   schema - <Object>, schema
+  //   category - <string>, schema name
+  //
+  // Returns: <this>
   definition(schema, category) {
     this.schema = schema;
     this.category = category;
     return this;
   }
 
-  copy(
-    // Copy references to new dataset
-    // Return: new Cursor instance
-  ) {
+  // Copy references to new dataset
+  //
+  // Returns: <Cursor>, new instance
+  copy() {
     throw new GSError(errorCodes.NOT_IMPLEMENTED);
   }
 
-  clone(
-    // Clone all dataset objects
-    // Return: new Cursor instance
-  ) {
+  // Clone all dataset objects
+  //
+  // Returns: <Cursor>, new instance
+  clone() {
     throw new GSError(errorCodes.NOT_IMPLEMENTED);
   }
 
-  enroll(
-    // Apply JSQL commands to dataset
-    jsql // commands array
-    // Return: previous instance
-  ) {
+  // Apply JSQL commands to dataset
+  //   jsql - <Array>, commands array
+  //
+  // Returns: <this>
+  enroll(jsql) {
     this.jsql = this.jsql.concat(jsql);
     return this;
   }
 
-  empty(
-    // Remove all instances from dataset
-    // Return: previous instance from chain
-  ) {
+  // Remove all instances from dataset
+  //
+  // Returns: <this>
+  empty() {
     throw new GSError(errorCodes.NOT_IMPLEMENTED);
   }
 
-  from(
-    // Synchronous virtualization converts Array to Cursor
-    // eslint-disable-next-line no-unused-vars
-    arr // array or iterable
-    // Return: new Cursor instance
-  ) {
+  // Synchronous virtualization converts Array to Cursor
+  //   arr - <Array>, array | iterable
+  //
+  // Returns: <Cursor>, new instance
+  from(arr) {  // eslint-disable-line no-unused-vars
     throw new GSError(errorCodes.NOT_IMPLEMENTED);
   }
 
-  map(
-    // Lazy map
-    fn // map function
-    // Return: previous instance from chain
-  ) {
+  // Lazy map
+  // fn - <Function>, map function
+  //
+  // Returns: <this>
+  map(fn) {
     this.jsql.push({ op: 'map', fn });
     return this;
   }
 
-  projection(
-    // Declarative lazy projection
-    metadata // projection metadata array of field names
-    // or structure: [ { toKey: [ fromKey, functions... ] } ]
-    // Return: previous instance from chain
-  ) {
+  // Declarative lazy projection
+  //   metadata - <string[]> | <Object>, projection metadata array of field names
+  // or object with structure: { toKey: [ fromKey, functions... ] }
```

Comment:
```suggestion
  //       or object with structure: { toKey: [ fromKey, functions... ] }
```

Reply:
length of the line 79 is 82. May be I can write this way:

```
  //   metadata - <string[]> | <Object>, projection metadata array of
  //       field names or object with structure:
  //       { toKey: [ fromKey, functions... ] }
  //

```